main/src/main/scala/sbt/Defaults.scala

```diff
@@ -3344,7 +3345,12 @@ object Classpaths {
         p1.extra(SbtPomExtraProperties.VERSION_SCHEME_KEY -> x)
       case _ => p1
     }
-    p2
+    val p3 = releaseNotesURL.value match {
+      case Some(u) =>
+        p1.extra("info.releaseNotesUrl" -> u.toExternalForm)
```

Comment:
```suggestion
        p2.extra("info.releaseNotesUrl" -> u.toExternalForm)
```
I think this must reference `p2` here.